autopush/router/apnsrouter.py

```diff
@@ -133,7 +133,7 @@ def _route(self, notification, router_data):
                 payload["cryptokey"] = notification.headers["crypto_key"]
             elif "encryption_key" in notification.headers:
                 payload["enckey"] = notification.headers["encryption_key"]
-            payload['aps'] = {"content-available": 1}
+            payload['aps'] = router_data.get('aps', {"content-available": 1})
```

Comment:
Yes.
There needs to be an "aps" block, thus why I default to `"{content-available": 1}`, however, 'content-available is equivalent to our silent notifications, so it can be problematic. I presume that the folks providing the override `aps` data are knowledgeable and allow for full customization.